era5cli/fetch.py

```diff
@@ -267,6 +280,34 @@ def _check_variable(self, variable):
                 "Invalid variable name: {}".format(variable)
             )
 
+    def _check_area(self):
+        """Confirm that area parameters are correct."""
+        (N, W, S, E) = self.area
+        if not (-90 <= N <= 90
+                and -90 <= S <= 90
+                and -180 <= W <= 180
+                and -180 <= E <= 180
+                and N > S
+                and W != E
```

Comment:
I'm surprised as I'd also expect `W < E`, but indeed the online CDS checker just accepts that. Does that lead to unexpected filenames?

And maybe we can issue a warning in this case (instead of an error)? Something like: "Be aware, this will probably work but we recommend changing it to avoid confusion"?

Reply:
I assumed that W-E was different, because they are the western and eastern coordinates of a grid that wraps around the world. You could  e.g. have a grid from 120W to -120E to get the pacific ocean, or from -120W to 120E to get most continents.
The filenames are probably confusing if you would read them as degrees on the map, which is what W-E-N-S may do, rather than numeric coordinates of the globe, with the W/E/N/S indicating the borders of the map you have chosen.

I can't even express this properly here, let alone in a filename :/ really not sure what is the best approach...

Reply:
Perhaps instead of W/E/N/S we could work with Left/Right/Top/Down, as that is the information we need.